build.gradle

```diff
@@ -15,12 +16,16 @@ node.nodeVersion = '10.5.0'
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
+jacoco {
+    toolVersion = "0.8.3"
```

Comment:
same here, place the plugins configuration together, according to the arrangement at the top.